app/javascript/components/index.js

```diff
@@ -94,6 +94,15 @@ export const coreComponents = [
       fetchTransformationPlansUrl:
         '/api/service_templates/?' +
         "filter[]=type='ServiceTemplateTransformationPlan'" +
+        '&filter[]=deleted_on=nil' +
```

Comment:
If I read @agrage's email correctly... "You shouldn't need to use the deleted on attribute directly, there is an active and an archived scope that you should be able to use."... this doesn't make sense?

@agrare can you confirm these two query filters for archived/unarchived are correctly constructed?

Reply:
@priley86 I can confirm that the following works:
```
>> ServiceTemplate.active
=> #<ActiveRecord::Relation [#<ServiceTemplate id: 1, name: nil, description: nil, guid: "ffd1a950-a007-49d8-891f-9c4741c5dbb9", type: nil, service_template_id: nil, options: {}, created_at: "2018-06-01 13:07:25", updated_at: "2018-06-01 13:07:25", display: nil, evm_owner_id: nil, miq_group_id: 1, service_type: "unknown", prov_type: nil, provision_cost: nil, service_template_catalog_id: nil, long_description: nil, tenant_id: 1, generic_subtype: nil, deleted_on: nil>]>
>> ServiceTemplate.archived
=> #<ActiveRecord::Relation []>
```
I can't confirm how that should be accessed through the API

Reply:
@agrare To be able to use `ServiceTemplate.archived` in an API, we would need `archived` to be a virtual column I think.
Accessing `deleted_on` directly is working well in the API, and should be OK to use it directly since there are no other complexities involved with the `archived`/ `unarchived` logic.

`deleted_on` == `nil` is `unarchived`
`deleted_on` != `nil` is `archived`

Reply:
Oh really having it as a scope isn't enough?  Well if you can get it working through the API and put a PR to core in I'll merge it.

Reply:
@agrare I just created this core PR - https://github.com/ManageIQ/manageiq/pull/17509

Reply:
@agrare Looks like we will need a PR to amend the virtual column to `archived`, 
since `archived?` works for this case -

`/api/service_templates/<id>?attributes=archived?`

but not for this case -

`/api/service_templates?filter[]=archived?=false`, what @michaelkro is using above

(There seems to be a bug with `filter` in the API... not sure how easy it is to resolve. From what I can tell it's related to `MiqExpression`)